test/specs/trailApexReplayDebugger.e2e.ts

```diff
@@ -0,0 +1,209 @@
+import {
+  InputBox,
+  QuickOpenBox,
+  TextEditor
+} from 'wdio-vscode-service';
+import {
+  TestSetup
+} from '../testSetup';
+import * as utilities from '../utilities';
+
+/**
+ * This test suite walks through the same steps performed in the "Find and Fix Bugs with Apex Replay Debugger" Trailhead Module;
+ * which can be found with the following link:
+ * https://trailhead.salesforce.com/content/learn/projects/find-and-fix-bugs-with-apex-replay-debugger
+ */
+describe('"Find and Fix Bugs with Apex Replay Debugger" Trailhead Module', async () => {
+  let prompt: QuickOpenBox | InputBox;
+  let testSetup: TestSetup;
+  const fiveMinutes = 5 * 60;
+
+  step('Set up the testing environment', async () => {
+    testSetup = new TestSetup('TrailApexReplayDebugger', false);
+    await testSetup.setUp();
+
+    // Create Apex class AccountService
+    await utilities.createApexClassWithBugs();
+
+    // Push source to org
+    const workbench = await browser.getWorkbench();
+    await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Push Source to Default Org and Override Conflicts', 5);
+  });
+
+  step('Run Apex Tests', async () => {
+    // Run SFDX: Run Apex tests.
+    const workbench = await browser.getWorkbench();
+    await utilities.runCommandFromCommandPrompt(workbench, 'View: Clear Output', 1);
+    prompt = await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Run Apex Tests', 1);
+
+    // Select the "AccountServiceTest" file
+    await prompt.selectQuickPick('AccountServiceTest');
+
+    // Wait for the command to execute
+    await utilities.waitForNotificationToGoAway(workbench, 'Running SFDX: Run Apex Tests', fiveMinutes);
+    await utilities.waitForNotificationToGoAway(workbench, 'Running SFDX: Run Apex Tests: Listening for streaming state changes...', fiveMinutes);
+    await utilities.waitForNotificationToGoAway(workbench, 'Running SFDX: Run Apex Tests: Processing test run', fiveMinutes, false);
+
+    const successNotificationWasFound = await utilities.notificationIsPresent(workbench, 'SFDX: Run Apex Tests successfully ran');
+    expect(successNotificationWasFound).toBe(true);
+
+    // Verify test results are listed on vscode's Output section
+    const outputPanelText = await utilities.attemptToFindOutputPanelText('Apex', '=== Test Results', 10);
+    expect(outputPanelText).not.toBeUndefined();
+    expect(outputPanelText).toContain('Assertion Failed: incorrect ticker symbol');
+    expect(outputPanelText).toContain('Expected: CRM, Actual: SFDC');
+  });
+
+  step('Set Breakpoints and Checkpoints', async () => {
+    // Get open text editor
+    const workbench = await browser.getWorkbench();
+    const editorView = workbench.getEditorView();
+    const textEditor = await editorView.openEditor('AccountService.cls') as TextEditor;
+    await textEditor.moveCursor(8, 5);
+
+    // Run SFDX: Toggle Checkpoint.
+    prompt = await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Toggle Checkpoint', 1);
+
```

Comment:
How about adding a check here to see if the checkpoint is present?
The CSS class is `'.codicon-debug-breakpoint-conditional'`

In a browser one would do:
```
const breakpoints = document.querySelectorAll('.codicon-debug-breakpoint-conditional');
```
...and then verify that `breakpoints.length === 1`

The syntax for wdio/selenium is a little different. (the global `document` object isn't available)

I'm not 100% positive, but I think something like:
```
const breakpoints = $$('.codicon-debug-breakpoint-conditional');
expect(breakpoints.length).toEqual(1);
```